app/src/main/res/values-de/strings.xml

```diff
@@ -27,6 +27,9 @@
  	<string name="pref_incoming_data_title">Eingehende Daten: %1$d - %2$d</string>   	
  	<string name="pref_incoming_data_dialog_title">Eingehende Daten</string>   	
 	<string name="pref_incoming_data_summary">Spielraum der eingehenden Daten</string>
+	<string name="pref_data_transfer_title">Datentransfer: %1$d - %2$d</string>
```

Comment:
PLease put a <!-- Google Translated --> on top of all the ones you did that way please

Reply:
not sure what is meant by this. could you clarify?

Reply:
Oh my message get truncated I was saying put a comment of Google translated above each of these so others can help us review